fv3core/stencils/fv_dynamics.py

```diff
@@ -207,7 +208,7 @@ def post_remap(state, comm):
         )
 
 
-def wrapup(state, comm):
+def wrapup(state, cube: CubedSphereCommunicator):
```

Comment:
I would prefer the change from 'comm' to 'cube' be done to the whole stack in one separate PR, after discussing whether we want that change (sounded like you and Jeremy prefer it, I am less keen on it, but don't have a super strong objection).  But this is not a big deal, just introduces one more inconsistency in a sea of many. I think 'comm'  makes more sense because this model is not going to be anything but a cubed sphere, and cube would make more sense if we had multiple kinds of communicators present. You had suggested 'cube_comm', which is more clear it is a cubed sphere communicator, but that's a little long. I'm not going to block this PR on this point, just wanted to express my perspective. If 'cube' is popular I'm not going to stop it.